src/index.css

```diff
@@ -5,3 +5,47 @@
 body {
   font-family: "Open Sans", Arial, sans-serif;
 }
+.with-background::before {
```

Comment:
Create a separate `slick.css` file and import that only for the required component.

Reply:
Done, thanks